invenio_app_rdm/requests_ui/templates/semantic-ui/invenio_requests/record-deletion/index.html

```diff
@@ -0,0 +1,114 @@
+{# -*- coding: utf-8 -*-
+
+  This file is part of Invenio.
+  Copyright (C) 2016-2025 CERN.
+
+  Invenio is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+{% extends "invenio_requests/details/index.html" %}
+
+{% set active_dashboard_menu_item = 'requests' %}
+{% set active_community_header_menu_item = 'requests' %}
+
+{%- block request_header %}
+    {% set back_button_url = url_for("invenio_app_rdm_users.requests") %}
+  {% from "invenio_requests/macros/request_header.html" import inclusion_request_header %}
```

Comment:
minor/nit: I guess this is already in other places, but the fact that we import and `inclusion_request_header` Jinja macro from `invenio-requests` sounds wrong. We should have a "generic" header macro/template, that just renders the request title and use it here.